<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_menu</class>
 <widget class="QMainWindow" name="MainWindow_menu">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow_menu{
	border-image: url(:/new/prefix2/Picture/talkDialogBackgroud2.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Findwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Findwidget{border-image:url(:/new/prefix2/Picture/mainBackImg.jpg);}</string>
    </property>
    <widget class="QLineEdit" name="signEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="UsrName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>方正舒体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>我的名字</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="closeLayout"/>
    </widget>
    <widget class="QToolButton" name="Head1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayout">
     <item>
      <widget class="QLineEdit" name="FindEdit">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>291</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
